### DISCUSSION TITLE
Επιλογή ημέρας και ώρας για συμπληρωματική ΟΣΣ1

### DISCUSSION THREAD

**Discussion by epidrome** [sdy60/syllabus#156] [Polls]
Αντί για μια μεγάλη ΟΣΣ1 μπορούμε να κάνουμε μερικές μικρότερες, οι οποίες θα γίνουν πιο νωρίς και θα υποστηρίξουν την μελέτη και την εκπόνηση της ΓΕ1. Με δεδομένο ότι θα γίνουν περισσότερες από μια ΟΣΣ, μπορείτε να απουσιάσετε σε μια ή περισσότερες.

**Comment by KonstantinosZarkadoulas** [comment]
Διάρκεια αυτής της μικρότερης ΟΣΣ;

**Reply by epidrome** [reply]
1-2 ώρες, ανάλογα με την συμμετοχή σας, οπότε, αθροιστικά, όλες μαζί με την τυπική στις 20 Οκτ, θα είναι τουλάχιστον 4 ώρες
